src/mon/Monitor.cc

```diff
@@ -323,6 +325,11 @@ void Monitor::do_admin_command(string command, cmdmap_t& cmdmap, string format,
     start_election();
     elector.stop_participating();
     ss << "stopped responding to quorum, initiated new election";
+  } else if (command == "ops") {
```

Comment:
The MDS and OSD both us dump_ops_in_flight; we should be consistent.
(And they also have dump_historic_ops, which ought to be uninteresting here but might as well add for consistency.)